grpc/errors.go

```diff
@@ -13,8 +13,8 @@ import (
 )
 
 var (
-	errIncompleteRequest  = errors.New("Incomplete gRPC request message")
-	errIncompleteResponse = errors.New("Incomplete gRPC response message")
+	ErrIncompleteRequest  = errors.New("Incomplete gRPC request message")
```

Comment:
Since we never do explicit equality comparison on these error values, there's no need to export them beyond this package. We can just define convenience errors locally in the nonce service. In other words, I think the duplication of the error text is an okay tradeoff for not having an additional exported value potentially used across all packages. Removing that refactoring would also have the nice side effect of making this a smaller PR that's easier to review thoroughly.